en/editor/publish/publish-huawei-ohos.md

```diff
@@ -0,0 +1,117 @@
+
+#### Generate key and certificate request file
+
+In **DevEco Studio**, click **Build -> Generate Key and CSR** in the upper menu bar to generate a key (`.p12` file), and then generate a certificate request file (`.csr`). For details, please refer to the [Generate Key and Certificate Request File](https://developer.harmonyos.com/cn/docs/documentation/doc-guides/publish_app-0000001053223745#EN-CN_TOPIC_0000001154985553__section7209054153620) documentation.
+
+> **Note**: When specifying the file path, the file name needs to be added with the corresponding suffix, otherwise the submission will not be successful.
+
+#### Generate and download a digital certificate (`.cer` file)
+
+- Visit the [AppGallery Connect website](https://developer.huawei.com/consumer/cn/service/josp/agc/index.html), select **Users and Access -> Certificate Management**.
+
+  > **Note**: Currently, only invited developers can access the **Certificate Management** menu. To apply, please send the developer's name, application background, supported device type, whether there is a request for app listing, and Developer ID to `agconnect@huawei.com`. Huawei operations staff will arrange a docking staff within 1-3 working days . <br />The developer ID query method is: log in to the AGC website, click **My Project**, select any project, and find the `Developer ID` in **Project Settings -> General**.
+
+  ![provision profile](./publish-huawei-ohos/cer-file.png)
+
+- Click the **Add Certificate** button in the upper right corner to fill in the information. **Certificate Type** Choose **Release Certificate**/**Debug Certificate** as required. The certificate request file is the `.csr` file generated in **DevEco Studio** in the previous step.
+
+- After filling in the information, click **Submit**, a release/debugging certificate (`.cer` file) will be generated, and click the **Download** button behind the certificate to download it to the local.
+
+For more information, please refer to the [Apply for Digital Certificate](https://developer.huawei.com/consumer/cn/doc/distribution/app/agc-harmonyapp-debugharmonyapp#h1-1598336089667) documentation.
+
+#### Generate and download HarmonyAppProvision Profile file (`.p7b` file)
+
+- Enter [AppGallery Connect website](https://developer.huawei.com/consumer/cn/service/josp/agc/index.html), select **My Project**, select the HarmonyOS project and application created before .
+
+- Click the **HarmonyOS Application -> HAP Provision Profile Management** item on the left, and then click the **Add** button at the top right of the page to create a `.p7b` file.
+
+  > **Note**: Currently, only invited developers can access the **HarmonyOS App** menu. To apply, please send the developer's name, application background, supported device type, whether there is a request for app listing, and Developer ID to `agconnect@huawei.com`. Huawei operations staff will arrange a docking staff within 1-3 working days.
```

Comment:
This can be deleted, now no longer need to apply, please help to delete the Chinese document also, thank you.